src/components/Header/elements/Menu.js

```diff
@@ -0,0 +1,13 @@
+function Menu() {
+    return(
+        <nav className="menu">
+            <ul>
+                <li><a href="#" className="menu__element active">Home</a></li>
+                <li><a href="#" className="menu__element">Orders</a></li>
+                <li><a href="#" className="menu__element">FAQ</a></li>
+            </ul>
```

Comment:
Create an array and use `.map()` on that array

And you will add routing later on so it will be efficient to have it now
`
const menuItems = ['Home', 'Orders', 'FAQ']`